src/sass/components/_forms.scss

```diff
@@ -123,4 +124,72 @@
             margin-bottom: 12px;
         }
     }
+    &.address-form {
+        padding-bottom: 30px;
+        margin-top: 30px;
+        .form-group:last-child {
+            margin-bottom: 0;
+            button[type=submit] {
+                width: 160px;
+                margin-right: 10px;
+            }
+            button {
+                font-size: 1.6rem;
+            }
+            @include Responsive(XS) {
+                @include flex-direction(row)
+            }
+        }
+        .form-group:nth-of-type(9) {
+            margin-bottom: 22px;
+            @include align-items(center);
+            @include Responsive(XS) {
+                @include flex-direction(row);
+            }
+            label {
+                width: unset;
+                @include Responsive(XS) {
+                    margin-bottom: 0;
+                }
+            }
+        }
+        .form-group {
+            &:nth-of-type(2), &:nth-of-type(7) {
+                @include Responsive(XS) {
+                    label {
+                        width: unset;
+                    }
+                }
+            }
```

Comment:
Hạn chế sử dụng nth-of-type để style như này. Vì rất khó để kiểm soát, trong trường hợp thêm hoặc bớt đi cái gì thì sẽ lại phải chỉnh lại style. Nên đặt 1 class riêng dành cho việc label { width: unset; } này